astropy/__init__.py

```diff
@@ -276,13 +220,13 @@ def online_help(query):
     webbrowser.open(url)
 
 
-__dir_inc__ = ['__version__', '__githash__', '__minimum_numpy_version__',
```

Comment:
This is an API breakage but I am not sure why this was provided to begin with. Any advice?